site/src/docs/components/header/index.mdx

```diff
@@ -64,6 +64,12 @@ Coming soon!
 
 Coming soon!
 
+### With navigation language selector
+
+- Using the Header.NavigationLanguageSelector is optional. Use the navigation language selector to define site languages. 
+- Header has onDidChangeLanguage function, where user can define a callback function which gets the selected language key (value) as parameter.
```

Comment:
It's a common practice in HDS docs to highlight code props and such, so it might be a good change to put onDidChangeLanguage => `onDidChangeLanguage`